Core/include/Acts/Utilities/detail/Extendable.hpp

```diff
@@ -26,11 +24,11 @@ namespace Acts::detail {
 template <typename... extensions_t>
 struct Extendable {
   // clang-format off
-    static_assert(detail::all_of_v<std::is_default_constructible<extensions_t>::value...>,
+    static_assert((std::is_default_constructible<extensions_t>::value && ...),
```

Comment:
```suggestion
    static_assert((std::is_default_constructible_v<extensions_t> && ...),
```
and the instances below